services/orchestrator_supervisor.py

```
@@ -0,0 +1,221 @@
+        self._state.stalled_recovered += 1
+        # Actually enqueue the run via the FIFO queue.
+        try:
+            from services.orchestrator_queue import get_orchestrator_queue
+            queue = get_orchestrator_queue()
+            await queue.enqueue(
+                run_id,
+                orchestrator.execute,
+                run._request,
+                resume_run_id=run_id,
+            )
+        except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)